### ISSUE TITLE
Audio gets disabled only in Brave

### ISSUE THREAD

**Issue by Fooughhy** [brave/brave-browser#16029]
## Description 

The issue I am having is that the Brave Browser changes its own audio device as well as disable the audio completely while I am watching a video.

## Steps to Reproduce

It will be extremely difficult for me to give any guidance on the steps required to reproduce this, but:
   1. Open Brave Browser, and navigate to e.g youtube.com (or primevideo.com, which I've also tried)
   2. Play any video (preferably a playlist)
   3. If that first video has audio, allow it to finish and go to the next in the playlist
   4. There is now no audio on that video, or any video from there on

## Actual result:

Audio, seemingly randomly, gets disabled in the browser. It is on a per-process basis, as one tab can have working audio while another does not. The audio has also been disabled during video playback or if I pause and unpause the video. And, as I said before, the video has changed audio device during playback. It was coming from my speakers, and all of a sudden it had changed to my headphones, while I was lying in bed watching The Grand Tour.

## Expected result:
Audio plays on the default windows sound device, and does not get disabled at all.

## Reproduces how often: 

Eventually, every time I have opened the browser.

## Brave version (brave://version info)

V1.24.86
Windows 10 Pro, build 19042.985

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:
I have actually changed my motherboard+cpu during this time and the issue was there before, and is still there after. I also just uninstalled and reinstalled the drivers for the audio as well.

The little sound-icon **is** present on the tab, just to be clear. There is supposed to be sound.

And finally, I have Spotify open and that does produce sound. Games as well, and have not had issues with sound at all.

- Does the issue resolve itself when disabling Brave Shields?
No, and a private window does not help 
- Does the issue resolve itself when disabling Brave Rewards?
No
- Is the issue reproducible on the latest version of Chrome? 
I do not know

**Comment by ryanbr** [comment]
Could you test in Chrome? I use Sony WX1000MX4's and make the headphones the default. If the headphones are off or they auto poweroff, they'll chose another option in the list of sound outputs, this is handled by Windows here. Maybe test by just enabling one default sound output @Fooughhy 

![windows-sound](https://user-images.githubusercontent.com/1659004/119579254-1f5fe880-be12-11eb-94d3-8f2ce9880a90.png)

**Comment by Fooughhy** [comment]
Thank you for the reply. Unfortunately, this issue is not present on Chrome (or at least Chromium, which I used). 

I use wired headphones, which can not be turned off (use the headphone jack and not USB), so it isn't them. I also disabled the screen speakers, having only my headphones active and it still didn't work. The issue is also very specifically that the first video I view on youtube after I've started Brave works, but not the next.
